src/main/java/cinema/model/Ticket.java

```diff
@@ -0,0 +1,55 @@
+package cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "movieId")
```

Comment:
```suggestion
    @JoinColumn(name = "movie_id")
```
remember about the correct column naming